pybossa/jobs.py

```diff
@@ -1175,9 +1200,11 @@ def push_notification(project_id, **kwargs):
                                filters=filters)
 
 
-def mail_project_report(info, current_user_email_addr):
+def mail_project_report(info, email_addr):
```

Comment:
This is to send weekly project report. hence, can it be send_weekly_project_report similar to other weekly report functions. with changing this, loggging message would also need an update.

Reply:
this func is also executed when user requests project report on UI.

Reply:
gotcha, thanks